.github/workflows/label-analysis.yml

```diff
@@ -14,18 +14,31 @@ env:
   GH_REPO: ${{ github.repository }}
 
 jobs:
-  assign-team-label:
+  assign-team-and-release-note-check:
```

Comment:
Coupling both of these is weird - we're coupling an action (assign-team) to a check (release-note), which have nothing in common.

Reply:
I agree, I wanted not to run in another runner as they need the same context, but this is probably an unnecessary saving